pilosa/orm.py

```diff
@@ -47,9 +47,9 @@
 
 
 class TimeQuantum:
-    """Valid time quantum values for fields having support for that.
+    """Valid time quantum values.
 
-    * See: `Data Model <https://www.pilosa.com/docs/data-model/>`_
+    * See: `Data Model/Time Quantum <https://www.pilosa.com/docs/latest/data-model/#time-quantum/>`_
```

Comment:
Remove trailing slash to get anchor links to work

Reply:
updated